src/transformers/generation/configuration_utils.py

```diff
@@ -211,6 +211,14 @@ class GenerationConfig(PushToHubMixin):
             Minimum token probability, which will be scaled by the probability of the most likely token. It must be a
             value between 0 and 1. Typical values are in the 0.01-0.2 range, comparably selective as setting `top_p` in
             the 0.99-0.8 range (use the opposite of normal `top_p` values).
+        moment_p_exponent (`float`, *optional*, defaults to 2.0):
+            The exponent to which probabilities are raised when calculating the moment. Higher values emphasize the
+            tail of the distribution more and reduce the truncation threshold. Values closer to 1 increase the threshold
+            to approach 1 as this value approaches 1. The default value represents the "P-Less" method's second moment,
+            i.e. `moment_p_exponent=2.0`.
+        moment_p_alpha (`float`, *optional*, defaults to 1.0):
+            Scaling factor applied to the moment threshold. Values < 1 make filtering more selective, while values > 1
+            make it less selective.
```

Comment:
Suggestion: if `moment_p` needs these two parameters to be correctly set, let's consider a dictionary (or a tuple). That way, users can immediately know that the attributes are used together.

I am aware other techniques have separate args, due to poor foresight 😢